external/netstandard/netstandard.depproj

```diff
@@ -35,6 +35,12 @@
     <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.2" Include="System.Data.SqlClient">
       <Version>4.3.0</Version>
     </PackageReference>
+    <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.3" Include="System.Security.Cryptography.Cng">
```

Comment:
Correct. We need the ref assemblies so we can produce the PSNE lib binaries that are needed to satisfy the package validation requirement that all ref assemblies have a matching lib assembly.

I'll try removing the 1.4 reference and do a full clean/build.